adaptlearning/adapt_framework
# issue/953: make core resilient to theme

* added core templates to core from theme
* added foundation less to core from theme
* added icon fonts
* changed grunt to handle nested less folders in core
* changed grunt to handle fonts in core
* changed grunt to handle no theme folder
* added fix to accessibility-state to core

to test:
1. git clone into branch issue/953, or download the branch
1a. remember to do npm install and adapt install on the framework
2. compile and check working
3. delete src/theme folder
4. recompile and check working (see ghastly 1990's no theme version)

#953
